ui/api/transport.go

```diff
@@ -300,6 +466,65 @@ func decodeGroupUpdate(_ context.Context, r *http.Request) (interface{}, error)
 	return req, nil
 }
 
+func parseSubtopic(subtopic string) (string, error) {
+	if subtopic == "" {
+		return subtopic, nil
+	}
+
+	subtopic, err := url.QueryUnescape(subtopic)
+	if err != nil {
+		return "", errMalformedSubtopic
+	}
+	subtopic = strings.Replace(subtopic, "/", ".", -1)
```

Comment:
Why this replacement with `.`? Can't you split on `/`?

Reply:
UI doesn't support subtopic for now, i will remove this function.